packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/aeERC20.sol

```diff
@@ -28,12 +28,12 @@ contract aeERC20 is ERC20PermitUpgradeable, ERC677Token {
     using AddressUpgradeable for address;
 
     function _initialize(
-        string memory name,
-        string memory symbol,
-        uint8 decimals
+        string memory name_,
```

Comment:
I think we should keep the underscores at the start (ie `_name`) for function parameters to keep it consistent.

Reply:
_name etc. shadows storage var declarations in ERC20; I think the convention generally worth sticking to is _var for internal variables, var_ for params